Author: Rashad Kanavath<rashad.kanavath@c-s.fr>
Last-Update: Thu, 17 Dec 2015 11:03:10 +0200
Description: Fix dh_shlibs reports missing symbols
--- a/CMake/FindOssim.cmake
+++ b/CMake/FindOssim.cmake
@@ -38,5 +38,5 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS( Ossim DEFAULT_MSG OSSIM_LIBRARY OSSIM_INCLUDE
 
 mark_as_advanced( OSSIM_INCLUDE_DIR OSSIM_LIBRARY )
 
-set(OSSIM_LIBRARIES ${OSSIM_LIBRARY})
+set(OSSIM_LIBRARIES "${OSSIM_LIBRARY};${OPENTHREADS_LIBRARY}")
 set(OSSIM_INCLUDE_DIRS ${OSSIM_INCLUDE_DIR})
