Description: Fix dh_shlibs reports missing symbols
Author: Rashad Kanavath <rashad.kanavath@c-s.fr>
Last-Update: 2015-12-17

--- otb.orig/CMake/FindOssim.cmake
+++ otb/CMake/FindOssim.cmake
@@ -55,5 +55,5 @@
 
 mark_as_advanced( OSSIM_INCLUDE_DIR OSSIM_LIBRARY )
 
-set(OSSIM_LIBRARIES ${OSSIM_LIBRARY})
+set(OSSIM_LIBRARIES "${OSSIM_LIBRARY};${OPENTHREADS_LIBRARY}")
 set(OSSIM_INCLUDE_DIRS ${OSSIM_INCLUDE_DIR})
